SPECS/fsverity-utils/fsverity-utils.spec

```diff
@@ -0,0 +1,116 @@
+Summary:        fsverity utilities
+Name:           fsverity-utils
+Version:        1.4
```

Comment:
Is there a reason that we're using an old version here? The latest release upstream seems to be `1.5`.

Reply:
It's ported from Fedora at the time. ` latest release upstream` do you refer to the source code https://git.kernel.org/pub/scm/fs/fsverity/fsverity-utils.git or a SPEC?

Reply:
I'm referring to the upstream repo you linked.

Reply:
Upgraded to 1.5